src/primitives/cuboid.js

```diff
@@ -12,15 +13,15 @@ const {geom3, poly3} = require('../geometry')
 const cuboid = (options) => {
   const defaults = {
     center: [0, 0, 0],
```

Comment:
this is odd, I had not noticed the addition of center to all these ?

Reply:
Yup. Center is part of all primitives except cylinders.